### PR TITLE
Implementation of Chaboche material model

### LINKED ISSUE #9: Implement "Chaboche"-type material model

### PR DESCRIPTION
**Pull request by jvaara**
After done, this pull request closes #9.

### CODE REVIEW

**Comment by jvaara** [comment]
image

File: examples/chaboche.jl [hunk 1]
```diff
@@ -103,14 +103,21 @@ function Chaboche(element, ip, time)
     b = element("b", ip, time)
 
     # Internal variables
-    stress = element("stress", ip, time)
-    plastic_strain = element("plastic strain", ip, time))
-    cumulative_equivalent_plastic_strain = element("cumulative equivalent plastic strain", ip, time)
-    backstress1 = element("backstress 1", ip, time))
-    backstress2 = element("backstress 2", ip, time))
-    yield_stress = element("yield stress", ip, time)
+    # stress = element("stress", ip, time)
```
Comment (ovainola) [inline examples/chaboche.jl]: Maybe just remove these lines, since they're not used

**Comment by jvaara** [comment]
image

**Comment by jvaara** [comment]
image

**Comment by ahojukka5** [comment]
Maybe add some tests?

**Pull request merged**